keg/db/dialect_ops.py

```diff
@@ -54,6 +54,21 @@ class PostgreSQLOps(DialectOperations):
     dialect_name = 'postgresql'
     option_defaults = {'schemas': ('public',)}
 
+    def create_all(self):
+        sql = []
```

Comment:
If I'm not mistaken, this and prep_empty() both create missing schemas with almost identical code.  Seems like you could put that logic in a class method like `def create_schemas()` and then call it from both locations, DRYing things up.

Reply:
:+1: I think we can create schemas with `IF NOT EXISTS` in either case.